beacon-chain/rpc/eth/validator/validator.go

```diff
@@ -671,7 +718,7 @@ func (vs *Server) PrepareBeaconProposer(
 	var validatorIndices []primitives.ValidatorIndex
 	newRecipients := make([]*ethpbv1.PrepareBeaconProposerRequest_FeeRecipientContainer, 0, len(request.Recipients))
 	for _, r := range request.Recipients {
-		f, err := vs.V1Alpha1Server.BeaconDB.FeeRecipientByValidatorID(ctx, r.ValidatorIndex)
+		f, err := vs.BeaconDB.FeeRecipientByValidatorID(ctx, r.ValidatorIndex)
```

Comment:
think we should have a check if beacondb is nil here? probably missed before.

Reply:
I think it's unnecessary. This should always be initialized.